src/bundle/Resources/public/js/scripts/sidebar/btn/location.edit.js

```diff
@@ -49,6 +48,10 @@
         attachModalListeners(wrapper);
     };
     const changeHandler = () => {
+        const checkedBtn = btns.find((btn) => btn.checked);
```

Comment:
What do you think about small optimization?
You can get the changed button from the event - `event.currentTarget`
Also, `languageCode` can be passed this to the `redirectToUserEdit` method.